src/lib/mdns/BUILD.gn

```diff
@@ -26,7 +26,12 @@ static_library("mdns") {
     "${chip_root}/src/lib/support",
   ]
 
-  sources = [ "Advertiser.h" ]
+  sources = [
+    "Advertiser.h",
+    "Helper.cpp",
+    "Helper.h",
```

Comment:
Please avoid using Helper/Support/Util until a last resort.

In this case I would suggest `ServiceNaming.h`

Reply:
Done